website/src/paragraph-panel.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { MdClose, MdNotes } from "react-icons/md"
+import { IconButton } from "./components"
+import { CollapsiblePanel, PanelSegment } from "./panel-layout"
+import * as css from "./panel-layout.css"
+import { TranslatedParagraph } from "./segment"
+
+const ParagraphPanel = (p: {
+  segment: TranslatedParagraph
+  setContent: (content: PanelSegment | null) => void
+}) => {
+  const concatSource = p.segment.source.reduce(
+    (paragraph, word) =>
+      `${paragraph} ${word.__typename === "AnnotatedForm" && word.source}`,
+    ""
+  )
+
+  return (
+    <>
+      <IconButton
+        className={css.wordPanelButton.basic}
+        onClick={() => p.setContent(null)}
+        aria-label="Dismiss selected paragraph information"
+      >
+        <MdClose size={32} />
+      </IconButton>
+
+      <header className={css.wordPanelHeader}>
+        <h1 className={css.noSpaceBelow}>
+          Selected
+          <br />
+          Paragraph
```

Comment:
Do you think we could show the paragraph number in the header? Same with the selected word. Instead of "selected word", "word #x"